src/modules/0.0.21/aws_appsync/entity/graphql_api.ts

```diff
@@ -0,0 +1,74 @@
+import { Column, Entity, PrimaryColumn } from 'typeorm';
+
+import { cloudId } from '../../../../services/cloud-id';
+
+export enum AuthenticationType {
+  AMAZON_COGNITO_USER_POOLS = 'AMAZON_COGNITO_USER_POOLS',
+  API_KEY = 'API_KEY',
+  AWS_IAM = 'AWS_IAM',
+  AWS_LAMBDA = 'AWS_LAMBDA',
+  OPENID_CONNECT = 'OPENID_CONNECT',
+}
+
+export enum DefaultAction {
+  ALLOW = 'ALLOW',
+  DENY = 'DENY',
+}
+
+@Entity()
+export class GraphqlApi {
+  @PrimaryColumn({
+    nullable: false,
+    type: 'varchar',
+  })
+  @cloudId
+  name: string;
+
+  @Column({
+    nullable: true,
+  })
+  apiId: string;
+
+  @Column({
+    nullable: true,
+  })
+  arn: string;
+
+  @Column({
+    type: 'enum',
+    enum: AuthenticationType,
+  })
+  authenticationType: AuthenticationType;
+
+  @Column({
+    type: 'json',
+    nullable: true,
+  })
+  lambdaAuthorizerConfig?: {
+    authorizerResultTtlInSeconds: number | undefined;
+    authorizerUri: string | undefined;
+    identityValidationExpression: string | undefined;
+  };
+
+  @Column({
+    type: 'json',
+    nullable: true,
+  })
+  openIDConnectConfig?: {
+    authTTL: number | undefined;
+    clientId: string | undefined;
+    iaTTL: number | undefined;
+    issuer: string | undefined;
+  };
```

Comment:
Nit. These two objects use `Ttl` and `TTL`. Can we standardize on one of the two?